app/src/main/java/com/example/tmdb/screens/Favorites.kt

```diff
@@ -0,0 +1,198 @@
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.scale
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.tmdb.R
+import com.example.tmdb.ui.theme.Blue
+import com.example.tmdb.Composables.MovieCard
+import com.example.tmdb.Composables.MovieItemViewState
+
+
+@Composable
+fun MovieList(
+    modifier: Modifier = Modifier,
+    onMovieItemClick: (MovieItemViewState) -> Unit = {},
+    MovieItems: List<MovieItemViewState>
+) {
+    LazyRow(
+        modifier = Modifier.fillMaxWidth(),
+        contentPadding = PaddingValues(15.dp)
+    ) {
+        items(MovieItems) {
+            MovieCard(
+                item = it,
+                onMovieItemClick = { onMovieItemClick(it) },
+                modifier = Modifier
+                    .size(
+                        width = dimensionResource(id = R.dimen.movie_card_width_favorite),
+                        height = dimensionResource(id = R.dimen.movie_card_height_favorite)
+                    )
+                    .padding(10.dp),
+            )
+        }
+    }
+}
+
+@Composable
+fun FavoritesScreen() {
+    var movies by remember {
+        mutableStateOf(
+            listOf(
+                MovieItemViewState(
+                    id = 1,
+                    title = "Iron Man 1",
+                    overview = "Overview",
+                    imageUrl = "R.drawable.iron_man_1"
+                ),
+                MovieItemViewState(
+                    id = 2,
+                    title = "Iron Man 1",
+                    overview = "Overview",
+                    imageUrl = "R.drawable.iron_man_1"
+                ),
+                MovieItemViewState(
+                    id = 3,
+                    title = "Iron Man 1",
+                    overview = "Overview",
+                    imageUrl = "R.drawable.iron_man_1"
+                )
+            )
+        )
+    }
+    val scaffoldStateMain: ScaffoldState = rememberScaffoldState()
+    Scaffold(
+        scaffoldState = scaffoldStateMain,
+        topBar = {
+            TopAppBar(
+                Modifier.background(Blue)
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.tmdb_sign),
+                    contentDescription = "Movie picture",
+                    alignment = Alignment.CenterStart,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .wrapContentWidth(align = Alignment.CenterHorizontally)
+                        .scale(3F)
+                )
+            }
+        },
+        bottomBar = {
+            BottomAppBar(
```

Comment:
You shouldn't have a bottom bar per screen. You should have following navigation:

* MainScreen - This is the only screen that hosts bottom bar.
* HomeScreen - First bottom bar item.
* FavoritesScreen - Second bottom bar item.
* DetailScreen - Shows on movie item click.